### ISSUE TITLE
Parallel multi-process in test.py

### ISSUE THREAD

**Issue by JetWings** [ShenghaiRong/BECO#8]
Parallel multi-process in test.py does not reduce CRF runtime. Setting n_jobs to 8 and 1 results in approximately equal running times.

**Comment by ShenghaiRong** [comment]
Please note that we report the COCO result without using the CRF post-process. Please remove "--crf" when evaluating the COCO dataset.

**Comment by JetWings** [comment]
Without using CRF, the test time remains unchanged between n_jobs=8 and 1.

**Comment by ShenghaiRong** [comment]
In our lab server, it takes about 40 minutes to evaluate the COCO val set using the offline logits.
![微信截图_20230808113559](https://github.com/ShenghaiRong/BECO/assets/18228614/7f8662c5-2790-44cf-99e3-1a7daacf8eb1)

**Comment by JetWings** [comment]
First of all, thank you for your response. What I meant is that parallel processing in test.py did not reduce the runtime. Setting n_jobs to both 8 and 1 resulted in the same number of CPU cores being utilized. However, I haven't been able to identify any issues in the code. I suggest you test with n_jobs=1 on your server.
